README.md

```diff
@@ -107,8 +111,9 @@ This will show you a log with details about errors (if any) and changes that wou
 ## 🔭 Future Features <a name="future-features"></a>
 
 - [x] **Create the mobile version of a new section to list all portfolio Projects**
-- [ ] **Create the mobile version of the contact section**
+- [x] **Create the mobile version of the contact section**
 - [ ] **Create the mobile version of a popup window to show the details of a project**
+- [ ] **Create the desktop version of the webpage**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
- [ ] To make your documentation be more professional and descriptive, kindly make sure all the `authors` contributors are present in the `Authors` section of your README